public/components/event_analytics/explorer/sidebar/sidebar.scss

```diff
@@ -124,3 +124,7 @@
 .ws__configPanel--right{
   overflow-y: unset;
 }
+
+.dscSidebar__item{
```

Comment:
Is this EUI react component class name? Or is it a custom class name? If it's EUI let's add another className to the element and use that. Overriding the EUI clas styles can influence other dashboard components too.